packages/react-icons/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spnels-fluent-system-test0000",
+  "version": "1.1.50",
+  "description": "Fluent System Icons are a collection of familiar, friendly, and modern icons from Microsoft.",
+  "author": "Microsoft Inc.",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-system-icons.git"
+  },
+  "scripts": {
+    "clean": "shx rm -rf ./components",
+    "cleanSvg": "shx rm -rf ./intermediate",
+    "copy": "node ../../importer/generate.js --source=../../assets --dest=./intermediate --extension=svg --target=react",
+    "convert": "npx @svgr/cli --no-prettier --template template.js --expand-props none --typescript --replace-attr-values \"#212121={props.primary}\" -d ./components ./intermediate/",
```

Comment:
Oh interesting, you are using SVGR. Just as an FYI, we had issues with this tool on a previous usage on the Fluid site due to groups on the SVG itself, but they were more complex shapes than icons. 

Do you have a way to view every icon once it's generated just to make sure svgr didn't cause any artifacts?

Reply:
Thanks. We don't have a visual regression test set up at the moment to test for that, but given the transforms happening and the simple svgs we build in the first place, we won't have any groups. If those were the breaking elements for you I feel we'll be good to go with svgr, but what did you all settle on using and how difficult would it be to swap in for svgr here?

Reply:
I ended up opening the SVG in Inkscape, and ungrouping everything to fix the issue and that worked. Also validated that if our designers gave us SVGs without being grouped, there were no artifacts. This was late 2019 so maybe the issue was fixed, but I thought it was worth mentioning as an FYI :)